README.md

```diff
@@ -31,21 +31,21 @@ Last 3 fields "httpRealm","usernameField","passwordField" filled empty because C
 
 > Minified version
 ```javascript
-var out="",out2="",pm=PasswordManager.getInstance(),model=pm.savedPasswordsList_.dataModel,pl=pm.savedPasswordsList_,version=loadTimeData.data_.browserVersion,timelag=model.length>100?15e3:5e3;for(document.getElementById("saved-passwords-list").scrollTop=document.getElementById("saved-passwords-list").scrollHeight,i=0;i<model.length;i++)chrome.send("requestShowPassword",[i]);setTimeout(function(){for(out2+="# Generated by Password Exporter; Export format 1.1; Encrypted: false\n",out2+='"hostname","username","password","formSubmitURL","httpRealm","usernameField","passwordField"',i=0;i<model.length;i++){var e=pl.getListItemByIndex(i),o=version.substring(8,10)>=51?model.array_[i].url:model.array_[i].origin;out+="\n"+o+"	"+model.array_[i].username+"	"+e.childNodes[0].childNodes[2].childNodes[0].value,out2+='\n"'+o+'","'+model.array_[i].username+'","'+e.childNodes[0].childNodes[2].childNodes[0].value.replace(/"/g,'""')+'","'+o+'"," "," "," "'}console.log(out),document.body.innerText=out2},timelag);
+var d=document,out="",out2="",pm=PasswordManager.getInstance(),pl=pm.savedPasswordsList_,model=pl.dataModel,version=loadTimeData.data_.browserVersion,timelag=model.length>100?15e3:5e3;for(d.getElementById("saved-passwords-list").scrollTop=d.getElementById("saved-passwords-list").scrollHeight,i=0;i<model.length;i++)chrome.send("requestShowPassword",[i]);setTimeout(function(){for(out2+="# Generated by Password Exporter; Export format 1.1; Encrypted: false\n",out2+='"hostname","username","password","formSubmitURL","httpRealm","usernameField","passwordField"',i=0;i<model.length;i++){var e=pl.getListItemByIndex(i),o=version.substring(8,10)>=51?model.array_[i].url:model.array_[i].origin;out+="\n"+o+"	"+model.array_[i].username+"	"+e.childNodes[0].childNodes[2].childNodes[0].value,out2+='\n"'+o+'","'+model.array_[i].username+'","'+e.childNodes[0].childNodes[2].childNodes[0].value.replace(/"/g,'""')+'","'+o+'"," "," "," "'}console.log(out),d.body.innerText=out2},timelag);
 ```
 
 > full version
 ```javascript
 var out = "";
 var out2 = "";
 var pm = PasswordManager.getInstance();
-var model = pm.savedPasswordsList_.dataModel;
 var pl = pm.savedPasswordsList_;
+var model = pl.dataModel;
 var version = loadTimeData.data_.browserVersion;
 var timelag = (model.length > 100) ? 15000 : 5000;
 document.getElementById("saved-passwords-list").scrollTop = document.getElementById("saved-passwords-list").scrollHeight;
 for (i = 0; i < model.length; i++) {
-	PasswordManager.requestShowPassword(i);
+	chrome.send('requestShowPassword', [i]);
```

Comment:
Align README.md with actual code. Please correct where is the actual version: here or in the js file?